### PR TITLE
first version of esProxy

### PR DESCRIPTION
**Pull request by awick**
@31453 let me know if there is anything you would like to change from a js standpoint

### CODE REVIEW

File: viewer/esProxy.js [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+  });
+}
+
+// Proxy
+
+function doProxy (req, res, cb) {
+  let result = '';
+  let url = elasticsearch + req.url;
+  console.log('URL', url);
+  var info = URL.parse(url);
```
Comment (31453) [inline viewer/esProxy.js:162]: Any reason you're using `var` in some places and `let` in others??

File: viewer/esProxy.js [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+
+// ============================================================================
+// handle the requests from the capture/viewer
+// ===========================================================================
+
+// Get requests
+app.get('*', (req, res) => {
+  const path = req.params['0'];
+
+  if (getExact[path]) {
```
Comment (31453) [inline viewer/esProxy.js:209]: What is this part all about? Why doesn't each case have anything inside of it? Is it a stub for the future?

Comment (31453) [reply viewer/esProxy.js:209]: And why not SWITCH!? 😄

Comment (31453) [reply viewer/esProxy.js:209]: Maybe create a function for the parts that are the same for the POST and GET parts?

**Review by 31453 — APPROVED** [review]

File: viewer/esProxy.js (hunk 1 above)
Comment (awick) [reply viewer/esProxy.js:162]: copy paste, fixing

File: viewer/esProxy.js (hunk 2 above)
Comment (awick) [reply viewer/esProxy.js:209]: they are empty because they aren't returning an error and will do success at bottom. I'll add a comment

**Review by 31453 — APPROVED** [review]

**Pull request merged**